src/routes.js

```diff
@@ -1,7 +1,14 @@
 import * as fs from 'fs';
 
+import { Storage } from '@google-cloud/storage';
+import multer from 'fastify-multer';
+
 import * as db from './db.js';
 
+const storage = new Storage({projectId: process.env.GCP_PROJECT_ID});
```

Comment:
This, and the relevant imports, should at some point become another module (video or storage, I'm not yet certain).